src/components/GordonDialogBox/index.jsx

```diff
@@ -31,48 +33,66 @@ const GordonDialogBox = ({
   children,
   ...otherProps
 }) => {
+  const [confetti, setConfetti] = useState(false); //added for the confetti- this, the popConfetti function, and div tag below are the same as the ones used for the happy birthday banner, just brought into this file
+
+  const popConfetti = () => {
+    setConfetti(true);
+    setTimeout(() => setConfetti(false), 5000);
+  };
+
   return (
-    <Dialog
-      className={styles.gc360_gordondialogbox}
-      {...otherProps}
-      aria-labelledby="alert-dialog-title"
-      aria-describedby="alert-dialog-description"
-    >
-      <DialogTitle
-        className={titleClass ? titleClass : styles.gc360_gordondialogbox_title}
-        id="alert-dialog-title"
+    <>
+      <div style={{ position: 'fixed', zIndex: 999999, top: -60, left: '50vw' }}>
+        <GordonConfetti active={confetti} colorOption="Gordon" />
+      </div>
+      <Dialog
+        className={styles.gc360_gordondialogbox}
+        {...otherProps}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
       >
```

Comment:
The GordonDialogBox component is used in a lot more places where we _do not_ want confetti to appear. I would suggest applying the confetti to the specific page in question, not here on this component.